include/swift/AST/DiagnosticsSema.def

```diff
@@ -3570,6 +3570,9 @@ ERROR(self_assignment_prop,none,
 ERROR(property_use_in_closure_without_explicit_self,none,
       "reference to property %0 in closure requires explicit use of 'self' to"
       " make capture semantics explicit", (Identifier))
+WARNING(warn_method_call_in_autoclosure_without_explicit_self,none,
+      "call to method %0 in autoclosure requires explicit use of 'self' to make"
```

Comment:
Since this seems to have gotten lost: I don't believe we should match the wording of the other errors since a warning cannot be construed as a requirement.